wordpress/wp-content/plugins/cds-base/classes/Modules/Forms/RequestSite/RequestSiteForm.php

```diff
@@ -155,12 +153,29 @@ public function render($atts, $content = null): string
                 ); ?>
             
                 <!-- start site -->
-                <?php echo Utils::textField(
-                    'site',
-                    __('English title of your site', 'cds-snc'),
-                    __('This title will appear at the top of your site. You can change this later.', 'cds-snc'),
-                    $all_values['site']
-                ); ?>
+                <div class="focus-group">
+                    <label class="gc-label" for="site" id="site-label">
+                        <?php _e('English title of your site', 'cds-snc'); ?>
+                    </label>
+                    <div id="site-desc" class="gc-description" data-testid="description">
+                        <?php _e('This title will appear at the top of your site. You can change this later.', 'cds-snc'); ?>
+                    </div>
+                    <input 
+                        type="text" 
+                        class="gc-input-text" 
+                        id="site" 
+                        name="site" 
+                        value="<?php echo $all_values['site']; ?>"
+                        aria-describedby="url-typer"
+                        required
+                    />
+                    <div id="url-typer" class="url-typer gc-description" aria-live="polite" aria-atomic="true">
+                        <div class="url-typer--empty"><?php _e('Enter a title to preview your URL'); ?></div>
+                        <div class="url-typer--message displayNone"><?php _e('Your url will be:'); ?> <strong>articles.alpha.canada.ca/<span id="url-typer__preview"></span></strong></div>
```

Comment:
Thinking we should update this to say

Your requested url will be --- or something like that .... in case the url is already taken.

Reply:
Changed it to say "Your URL preview:", pretty close to what we are saying in the 'empty' message.